multinet/api.py

```diff
@@ -100,6 +102,23 @@ def get_table_rows(workspace: str, table: str, offset: int = 0, limit: int = 30)
     return Workspace(workspace).table(table).rows(offset, limit)
 
 
+@bp.route("/workspaces/<workspace>/tables/<table>/metadata", methods=["GET"])
+@require_reader
+@swag_from("swagger/get_metadata.yaml")
+def get_table_metadata(workspace: str, table: str) -> Any:
+    """Retrieve the metadata of a table, if it exists."""
+    metadata = Workspace(workspace).table(table).get_metadata()
+    return "" if metadata is None else metadata.dict()
+
+
+@bp.route("/workspaces/<workspace>/tables/<table>/metadata", methods=["PUT"])
+@require_reader
+@swag_from("swagger/set_metadata.yaml")
+def set_table_metadata(workspace: str, table: str) -> Any:
+    """Retrieve the rows and headers of a table."""
+    return Workspace(workspace).table(table).set_metadata(request.json).dict()
```

Comment:
Again, what if the workspace/table isn't found

Reply:
Same answer as above